tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/inheritance_test.dart

```diff
@@ -126,18 +128,17 @@ void main() {
       var error = collector.errors.first;
       expect(
         error.message,
-        'The "extends" property is mutually exclusive with the "table" property.',
+        'You can only extend classes from your own project.',
       );
     });
 
     test(
-        'Given a child-class extends a class with a table defined, then collect an error that parent class cannot have a table',
+        'Given a child-class when inheritance is not enabled, then error is collected that the "extends" property is not allowed',
```

Comment:
Suggestion(NIT): You changed `child-class` to `child class` in the previous test. Should this be changed for all tests in this file? (And possibly other files)